src/Handler/CurlMultiHandler.php

```diff
@@ -20,11 +20,30 @@ class CurlMultiHandler
 {
     /** @var CurlFactoryInterface */
     private $factory;
+
     /** @var int */
     private $selectTimeout;
-    private $active;
+
+    /** @var resource|null the currently executing resource in `curl_multi_exec`. */
+    private $active = null;
```

Comment:
Don't need to initialise this. Already initialises to null?

Reply:
I remember making a change in a PHP7.4 environment, where "uninitialized exceptions" were thrown. See: https://stackoverflow.com/questions/60112627/how-to-let-phpunit-test-property-initialization-in-php7-4

Nikic recommends initializing as `null` when the constructor doesn't write a field's value.

Reply:
That only is relevant with typed properties.